README.md

```diff
@@ -0,0 +1,63 @@
+# Install Conan Action
+
+[![Validate Conan Action](https://github.com/uilianries/conan-github-action/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/uilianries/conan-github-action/actions/workflows/ci.yml)
+[![GitHub Marketplace](https://img.shields.io/badge/Marketplace-Conan%20Package%20Manager-blue.svg?colorA=24292e&colorB=0366d6&style=flat&longCache=true&logo=github)](https://github.com/marketplace/actions/conan-io-github-action)
+
+
+A GitHub Action to install and configure the Conan package manager in your workflow.
+This action provides a simple and flexible way to set up Conan with custom configurations and audit settings.
+
+## Features
+
+- 🚀 Install any version of Conan
+- ⚙️ Apply custom Conan configurations
+- 🔐 Configure audit tokens
+- 🗂️ Cache Conan packages using GitHub cache
+- 🔍 Support for custom Conan cache directory
```

Comment:
```suggestion
- 🔍 Support for custom Conan home
```

Reply:
Done, replaced by `🔍 Support for custom Conan home`